src/types/Patterns.ts

```diff
@@ -34,6 +34,7 @@ export const Patterns = {
     'i'
   ),
   UUID: /^[a-f0-9]{8}(-[a-f0-9]{4}){4}[a-f0-9]{8}$/i,
+  EXTID: /^[a-z0-9]{6,36}$/i,
```

Comment:
@oyo is this enough? UUIDs have "-" included. We need to ensure that "-" are allowed as well please

Reply:
@jjeroch further below this file the check is:
` Patterns.EXTID.test(expr) || isUUID(expr)`
so currently we either accept an alphnumeric string (digits and letters) lenght between 6 and 36 OR a UUID

Reply:
ah ok :) approved